### PR TITLE
feat(service-antenne): add component and query

### PR DESCRIPTION
**Pull request by kwiss**

### CODE REVIEW

File: packages/app/src/components-v2/Antennes/Antennes.js [hunk 1]
```diff
@@ -1,16 +1,47 @@
...
+  if (error) {
+    return (
+      <Card p="1">
+        <Box sx={{ position: "relative", p: "6" }}>error</Box>
+      </Card>
+    );
+  }
+
+  const { service_antenne } = data;
+  console.log(service_antenne);
```
Comment (douglasduteil) [inline packages/app/src/components-v2/Antennes/Antennes.js]: console.log

Comment (kwiss) [reply packages/app/src/components-v2/Antennes/Antennes.js]: WIP, sorry

File: packages/app/src/components-v2/ServiceMesures/ServiceAcceptMesure.js [hunk 2]
```diff
@@ -32,27 +35,36 @@ export const ServiceAcceptMesure = props => {
...
-                  ville: values.ville
-                }
+                  ville: values.ville,
+                  antenne_id: values.antenne_id.value
+                },
+                refetchQueries: ["mesures", "mesures_aggregate"]
               });
               setSubmitting(false);
               setPanelType(null);
               setCurrentMesure(null);
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceAcceptMesure.js]: should be moved to the proper async cb

File: packages/app/src/components-v2/ServiceMesures/ServiceAcceptMesure.js [hunk 3]
```diff
@@ -32,27 +35,36 @@ export const ServiceAcceptMesure = props => {
         <Formik
           onSubmit={(values, { setSubmitting }) => {
             setTimeout(() => {
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceAcceptMesure.js]: remove setTimeout

**Pull request merged**